tools/export/iar/iar_definitions.json

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
Do these names correspond with the `device_name` attribute in `hal/targets.json`?

Do you think it would add too much clutter to `hal/targets.json` if this data was added there instead of creating a separate file?